src/main.js

```diff
@@ -37,6 +37,31 @@ function createWindow() {
     icon: path.join(__dirname, 'assets/icons/png/256x256.png'),
   });
 
+  if (process.env.NODE_ENV === 'development') {
```

Comment:
While this is definitely a best practice for most apps, I actually think we should add the React devtools to the production build as well :)

The reason is that Guppy will be used exclusively by developers, and I like the idea of allowing them to peek under the hood / poke around with Guppy if they want. Could be educational.

I believe the React devtools work with the prod build (after all, I can use them on prod sites in Chrome), but I'm not sure about `REACT_PERF`. Possibly we'd only want to install that one in dev:

```js
const extensions = [REACT_DEVELOPER_TOOLS, ChromeLens];

if (process.env.NODE_ENV === 'development') {
  extensions.push(REACT_PERF
}
```

Reply:
it's different using an extension you already have installed on your browser and impacting the startup time of an app by downloading the extensions...
But if you feel like the startup time is less important than the ability to poke at the source, happy to make the change

Reply:
Does it redownload the extension each time (and not cache it)?

Reply:
No I think it caches it but still

Reply:
So long as the user's internet connection is decent (>1MB/s or so) it shouldn't add a relevant delay to startup, and even if it does add an extra few seconds, I think @joshwcomeau is right, knowing about React DevTools when starting to learn React is a HUGE boon for new developers - may even be worth pointing it out from within the app once they've started a CRA project.